test/integration/authenticatable_test.rb

```diff
@@ -557,6 +557,18 @@ class AuthenticationOthersTest < Devise::IntegrationTest
       ActiveSupport::Notifications.unsubscribe(subscriber)
     end
   end
+
+  test 'public action should return notification payload with 500 status' do
+    begin
+      subscriber = ActiveSupport::Notifications.subscribe /process_action.action_controller/ do |_name, _start, _finish, _id, payload|
+        assert_equal 500, payload[:status]
```

Comment:
This is basically a test case copied from the above one, but using a public action that raises an error (see rest of commit)

This assertion fails unexpectedly:
```
devise(master)% ruby -I"lib:test" test/integration/authenticatable_test.rb -n /public/

==> Devise.orm = :active_record
Expected string default value for '--orm'; got false (boolean)
Run options: -n /public/ --seed 25980

# Running:

F

Failure:
AuthenticationOthersTest#test_public_action_should_return_notification_payload_with_500_status [test/integration/authenticatable_test.rb:564]:
Expected: 500
  Actual: 401
```